GAE/src/org/akvo/flow/rest/ReportRestService.java

```diff
@@ -41,7 +45,10 @@
 @Controller
 @RequestMapping("/reports")
 public class ReportRestService {
-    private static final Logger log = Logger.getLogger(ReportRestService.class.getName());
+	@Autowired
```

Comment:
Just a small note here.. if we can dig a little deeper lets try to stay away from any @Autowired spring stuff.  We removed as much as possible of the spring auto setup to improve performance. More so if the main motivation is to know which protocol is used.